api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -222,6 +222,15 @@ private User initializeUserIfNeeded() {
       // Service accounts don't need further initialization.
       return user;
     }
+
+
+    if (user != null &&
```

Comment:
I can appreciate checking for null, but do we need it? Line 220 should give us a non-null object. Also, on lines 221 and 235 (directly around this one), we are not checking for null. Did you see this in practice? If we get to this point with a null user, then we might have a bigger problem and should probably throw an exception.